### PR TITLE
Add the 'left outer' to the schema and type enum for JoinStep.

### LINKED ISSUE #549: When I choose a left outer join type, I don't see an error message.
The `left outer` join type is not an accepted value by the join step.

It should be.

### PR DESCRIPTION
**Pull request by erenard**
Fixes #549 

Add `left outer` to the lists of join types.

### CODE REVIEW

File: src/components/stepforms/schemas/join.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default {
     },
     type: {
       type: 'string',
-      enum: ['left', 'inner'],
+      enum: ['left', 'inner', 'left outer'],
```
Comment (adimascio) [inline src/components/stepforms/schemas/join.ts:21]: Thanks for the fix!

I'm just wondering: shouldn't the `join.ts` schema file be imported in `JoinStepForm.vue` in order to make sure they're always sync?

Comment (erenard) [reply src/components/stepforms/schemas/join.ts:21]: Good suggestion, thanks.

File: src/components/stepforms/JoinStepForm.vue [hunk 2]
```diff
@@ -39,6 +39,7 @@
 import { Prop } from 'vue-property-decorator';
 
 import { StepFormComponent } from '@/components/formlib';
+import { default as JoinStepFormSchema } from '@/components/stepforms/schemas/join.ts';
```
Comment (adimascio) [inline src/components/stepforms/JoinStepForm.vue:42]:
```suggestion
import JoinStepFormSchema from '@/components/stepforms/schemas/join.ts';
```

**Comment by adimascio** [comment]
[As a conceptor, i would like to be able to do a "left outer" join in the VQB](https://trello.com/c/uerXPvlf/4131-as-a-conceptor-i-would-like-to-be-able-to-do-a-left-outer-join-in-the-vqb)

**Pull request merged**